File: Homework1/Arrays/Assets/stackedColors.js
var aP:Array = new Array(9);

function Start () {	for(var i:int = 0; i < 9; i++)	{		aP[i] = GameObject.CreatePrimitive(PrimitiveType.Cube);		aP[i].transform.position.y = i + 1;	}}

function Update () {
	for(var i:int = 0; i < 9; i++)
	{
		aP[i].renderer.material.color = 
										new Color(Random.Range(0.0,1.0),
										Random.Range(0.0,1.0),
										Random.Range(0.0,1.0));
	}}

File: Homework1/Arrays/Assets/stackedColors.cs
using UnityEngine;
using System.Collections;

public class stackedColors : MonoBehaviour 
{
    GameObject[] Objects = new GameObject[9];

	void Start ()
	{
        for (int i = 0; i < Objects.Length; i++)
        {
            Objects[i] = GameObject.CreatePrimitive(PrimitiveType.Cube);
            Objects[i].transform.Translate(0, i + 1f, 0);
        }
	}
    
    void Update()
	{
        foreach (GameObject o in Objects)
            o.renderer.material.color = new Color(Random.RandomRange(0f, 1f), Random.RandomRange(0f, 1f), Random.RandomRange(0f, 1f));
	}
}
